example/spec/parallel/kinesisTests/LambdaEventSourceSpec.js

```diff
@@ -149,7 +155,7 @@ describe('When adding multiple rules that share a kinesis event stream', () => {
           recordIdentifier,
           workflowArn,
           maxWaitForSFExistSecs,
-          2
+          1
```

Comment:
Why did this value change?

Reply:
This test is really only expecting 1 workflow to be started for the one enabled rule, but the value of `2` meant that the helper kept looking for another workflow until the timeout for `waitForAllTestSfForRecord` expired. So it was making the test take longer for no reason.

Reply:
Oh 😯  Thank you for fixing this and for the explanation!